service/controller/resource/tccpnoutputs/create.go

```diff
@@ -0,0 +1,139 @@
+package tccpnoutputs
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/aws-operator/service/controller/controllercontext"
+	"github.com/giantswarm/aws-operator/service/controller/internal/cloudformation"
+	"github.com/giantswarm/aws-operator/service/controller/key"
+)
+
+const (
+	DockerVolumeResourceNameKey   = "DockerVolumeResourceName"
+	HostedZoneNameServersKey      = "HostedZoneNameServers"
+	MasterImageIDKey              = "MasterImageID"
+	MasterInstanceResourceNameKey = "MasterInstanceResourceName"
+	MasterInstanceTypeKey         = "MasterInstanceType"
+	OperatorVersion               = "OperatorVersion"
+	VPCIDKey                      = "VPCID"
+	VPCPeeringConnectionIDKey     = "VPCPeeringConnectionID"
+)
```

Comment:
There are a couple of things that the outputs of the TCCPN stack do not define. So this needs a little check.